### PR TITLE
Update to Vanilla 3.0.1

### PR DESCRIPTION
**Pull request by bartaz**
## Done

- Updated to 3.0.1
- Removed neutral button
- Updated Chips to their new HTML format (button)
- replaced node-sass with sass

## QA

- Make sure everything works as expected, review different components, especially the ones affected by 3.0
- https://react-components-656.demos.haus/?path=/story/accordion--default-story

### CODE REVIEW

**Comment by bartaz** [comment]
Demo: https://react-components-656.demos.haus/?path=/story/accordion--default-story

File: src/components/ActionButton/__snapshots__/ActionButton.test.tsx.snap [hunk 1]
```diff
@@ -5,7 +5,7 @@ exports[`ActionButton matches loading snapshot 1`] = `
   loading={true}
 >
   <button
-    className="p-action-button p-button--neutral is-processing"
+    className="p-action-button p-button--undefined is-processing"
```
Comment (bethcollins92) [inline src/components/ActionButton/__snapshots__/ActionButton.test.tsx.snap:8]: I don't think this should be undefined

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
QA LGTM, just the query on the `p-button--undefined` class in the snapshot

**Review by bethcollins92 — APPROVED** [review]
Go Go goooooo

**Pull request merged**